pkgs/by-name/ta/tacent/package.nix

```
@@ -0,0 +1,37 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tacent";
+  version = "0.8.18";
+
+  src = fetchFromGitHub {
+    owner = "bluescan";
+    repo = "tacent";
+    rev = "refs/tags/${version}";
```

```suggestion
    tag  = version;
```
Newly introduced `tag` argument.